src/components/Inputs/RadioButtons/RadioButtons.js

```diff
@@ -0,0 +1,293 @@
+  DOWN: 40,
+}
+
+// Wrapping this way facilitates spying with spyOn
+export const focusHelper = {
+  focus: (elementRef) => {
+    elementRef.current.focus()
+  },
+}
+
+function RadioButton({
+  name,
+  value,
+  checked,
+  tabIndex,
+  required,
+  disabled,
+  label,
+  onClick,
+  onChange,
+  ...rest
+}) {
+  const [, includesKeysOrThrow] = useIncludes(['name', 'label'])
+  includesKeysOrThrow({ name, label })
+  const [includesInvalid] = useInvalid(Object.keys(RadioButton.propTypes))
+  includesInvalid(rest)
+
+  const spanRadio = useRef(null)
+
+  // This allows the user to arrow navigate within radio group preserving focus affordance
+  useEffect(() => {
+    if (checked) {
+      focusHelper.focus(spanRadio)
+    }
+  }, [checked])
+
+  return (
+    <label className={styles.RadioButton}>
+      <span
+        role="radio"
+        ref={spanRadio}
+        value={value}
+        aria-checked={checked}
+        tabIndex={tabIndex}
+      >
+        <input
+          type="radio"
+          name={name}
+          value={value}
+          checked={checked}
+          onClick={onClick}
+          onChange={onChange}
+          required={required}
+          disabled={disabled}
+          data-tid={rest['data-tid']}
+        />
+        <aside />
+      </span>
+      <Body.Regular400 color={COLORS.GRAY_PRIMARY}>{label}</Body.Regular400>
+    </label>
+  )
+}
+
+RadioButton.propTypes = {
+  name: PropTypes.string.isRequired,
+  value: PropTypes.string.isRequired,
+  checked: PropTypes.bool,
+  tabIndex: PropTypes.number,
+  required: PropTypes.bool,
+  disabled: PropTypes.bool,
+  label: PropTypes.node.isRequired,
+  'data-tid': PropTypes.string,
+  onClick: PropTypes.func,
+
+  // These will appear if RadioButtonGroup is used with redux-form:
+  onBlur: PropTypes.func,
+  onChange: PropTypes.func,
+  onDragStart: PropTypes.func,
+  onDrop: PropTypes.func,
+  onFocus: PropTypes.func,
+}
+
+export function RadioButtonGroup({
+  options,
+  onChange,
+  formChangeHandler,
+  allCaps = true,
+  name = `radio-button-group-${uuidv4()}`,
+  initialValue = undefined,
+  currentValue,
+  currentError,
+  formTouched,
+  disabled,
+  validator,
+  required,
+  labelCopy,
+  ...rest
+}) {
+  let initialSelected
+  if (currentValue || typeof currentValue === 'boolean') {
+    initialSelected = currentValue
+  } else if (initialValue || typeof initialValue === 'boolean') {
+    initialSelected = initialValue
+  }
+
+  const [selectedValue, setSelectedValue] = useState(initialSelected)
+  const [isAnswered, setIsAnswered] = useState(false)
+  const resolvedValidator = validator ? validator : () => ''
+  const [getError, setError, , validate] = useErrorMessage(resolvedValidator)
+
+  const validationRadio = () => {
+    let errorMessage = validate(selectedValue)
+    errorMessage = errorMessage.length ? errorMessage : ''
+    setError(errorMessage)
+    if (formChangeHandler) {
+      // Update form with the new value and a falsy error message
```

Comment:
why falsy? can't errorMessage be set? what am I missing ?

Reply:
That's just a poor leftover comment. I'll remove it